resources/assets/js/components/JobDetails/JobDetails.tsx

```diff
@@ -253,44 +327,34 @@ const JobDetails: React.FunctionComponent<
       )
       .required(intl.formatMessage(validationMessages.required)),
     level: Yup.number()
-      .min(1, intl.formatMessage(validationMessages.invalidSelection))
+      .min(1)
```

Comment:
Any particular reason the message was removed from this one?

Reply:
Nope! Not sure how that happened.